src/System.Linq.Expressions/src/Resources/Strings.resx

```diff
@@ -337,7 +337,7 @@
     <value>Incorrect number of arguments for the given members </value>
   </data>
   <data name="LambdaTypeMustBeDerivedFromSystemDelegate" xml:space="preserve">
-    <value>Lambda type parameter must be derived from System.Delegate</value>
+    <value>Lambda type parameter must be derived from System.MulticastDelegate (and not System.MulticastDelegate itself)</value>
```

Comment:
I'm finding this grammar confusing.  I'd suggest just:
```
Lambda type parameter must be derived from System.MulticastDelegate.
```
or if you feel the parenthetical is important, I think the word "be" is missing... otherwise it sounds to me like it's saying "it must be derived from A and not from A", which is confusing.